src/cljs/rems/status_modal.cljs

```diff
@@ -47,6 +47,55 @@
        (when @state [component @state])
        [:button.btn.btn-secondary {:on-click #(reset! state (assoc opened-state :on-close on-close))} "Open modal"]])))
 
+(defn status-modal-opts
+  "Returns a map of modal options, that can be used by status-modal/situational-status-modal.
+
+   The returned value is a map containing various event handlers, and most importantly,
+   a state atom under the key :state-atom."
+  [{:keys [on-success on-pending on-error on-close-after-success on-close-after-error] :as modal-options}]
+  (let [state (r/atom nil)]
+    (merge
+     modal-options
+     {:state-atom state
+      :on-pending #(do
+                     (swap! state assoc :status :pending)
+                     (when on-pending (on-pending)))
+      :on-success #(do
+                     (swap! state assoc :status :saved)
```

Comment:
`:saved` could now be named `:success`

Reply:
Both are now supported. `:saved` is used  in so many places, I don't dare to change it everywhere.